### PR TITLE
Mark old issues as stale

### PR DESCRIPTION
**Pull request by CGNonofr**
and some minor fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,6 +364,10 @@ It includes:
 From CLI run:
 
 ```bash
+# build monaco-vscode-api (the demo use it as a local dependency)
+npm ci
```
Comment (kaisalmen) [inline README.md:368]: You should also mention that the project does not properly build on Windows. You require Linux (including WSL) or Mac.

**Review by kaisalmen — APPROVED** [review]
The comment is not blocking. We should add that later or now, LGTM.

File: README.md (hunk 1 above)
Comment (CGNonofr) [reply README.md:368]: Good idea! 500285db26ca5ca74021a0aa95ca447d0cf17490

File: README.md [hunk 2]
```diff
@@ -380,6 +380,8 @@ For the debug feature, also run:
 npm run start:debugServer
 ```
 
+⚠️ Building monaco-vscode-api is only supported on Linux or Mac. It you use Windows, have a look at [WSL](https://learn.microsoft.com/fr-fr/windows/wsl/install) ⚠️
```
Comment (kaisalmen) [inline README.md:383]: You linked the french page. It does it automatically when going to the page. I get the german page.
Please use this:
https://learn.microsoft.com/en-us/windows/wsl/install

Comment (CGNonofr) [reply README.md:383]: Oh well spotted

Comment (kaisalmen) [reply README.md:383]: I though: "Oh la la, je ne comprends pas" 😆

Comment (CGNonofr) [reply README.md]: e09e563aad1e68d1723aebc209e6afc5d272d94b

**Review by kaisalmen — APPROVED** [review]
Let's go

**Pull request merged**